system_tests/espresso-e2e/genesis/demo.toml

```diff
@@ -0,0 +1,12 @@
+[stake_table]
```

Comment:
This file has been included in the docker image and it seems unnecessary to have it here

Reply:
Sorry. Yes we should add this into the repo. Becasue the genesis file contains some other information